todo_app/trello/trello_item.py

```diff
@@ -29,4 +48,49 @@ def status(self):
   def status(self,value):    
     self._status = value
 
+  @classmethod
```

Comment:
Is there a reason to prefer class methods here, rather than just a method or property on the instance? You don't use the cls parameter and pass through an instance of a trello_item, which suggests it would make more sense as a standard function/property receiving `self`

Reply:
These cls methods are passed as a parameter to TrelloViewModel filter_items method. This encapsulates the knowledge of the status of the TrelloItem within the TrelloItem class. 

Is there another better way to pass a function pointer into a method?